changelog/updates/2022-09-14-intel-microcode.md

```diff
@@ -0,0 +1 @@
+- intel-microcode ([20220809_p20220809](https://github.com/intel/Intel-Linux-Processor-Microcode-Data-Files/releases/tag/microcode-20220809))
```

Comment:
How about:

```suggestion
- intel-microcode ([20220809](https://github.com/intel/Intel-Linux-Processor-Microcode-Data-Files/releases/tag/microcode-20220809))
```

The `_p<NUMBER>` part is used for some Gentoo-specific collection stuff which we don't use, because we enable vanilla USE flag.